concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1258,6 +1232,44 @@ jobs:
 
                   cf push
 
+      - task: deploy-healthcheck
```

Comment:
For the sake of consistency and to  make it more clear, shall we move this to a dedicated separated file in tasks? This way we can have a `paas-cf/concourse/tasks/remove-healthcheck-app.yml` and a `paas-cf/concourse/tasks/push-healthcheck-app.yml`